libraries/AdaptiveExpressions/Expression.cs

```diff
@@ -440,7 +439,9 @@ public IReadOnlyList<string> References()
                     var iteratorName = (string)(children[1].Children[0] as Constant).Value;
 
                     // filter references found in children 2 with iterator name
+#pragma warning disable CA1309 // Use ordinal stringcomparison
                     var nonLocalRefs2 = refs2.Where(x => !(x.Equals(iteratorName) || x.StartsWith(iteratorName + '.') || x.StartsWith(iteratorName + '[')))
```

Comment:
Why not just use ordinal string comparison?

Reply:
fixed after merged master